File: DAY-6/solution.py
a=(int(input("Enter a: ")))
b=(int(input("Enter b: ")))
min=0
gcd=1
if a<b:
    min = a
else:
    min = b
for i in range(1, min + 1):
    if a % i == 0 and b % i == 0:
        gcd = i
print("GCD =", gcd)

File: DAY-6/solution.java
import java.util.Scanner;
public class solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a and b: ");
        int a = sc.nextInt();
        int b = sc.nextInt();
        int min;
        int gcd = 1;
        if (a < b) {
            min = a;
        } else {
            min = b;
        }
        for (int i = 1; i <= min; i++) {
            if (a % i == 0 && b % i == 0) {
                gcd = i;
            }
        }
        System.out.println("GCD = " + gcd);
        sc.close();
    }
}